### ISSUE TITLE
question about FeatureFlowAttention: flow.detach()

### ISSUE THREAD

**Issue by mansang127** [haofeixu/gmflow#18]
Code snippets:
flow = self.feature_flow_attn(feature0, flow.detach(),
                              local_window_attn=prop_radius > 0,
                              local_window_radius=prop_radius)

I'm curious about the flow.detach(). What was your idea for that design? I guess multipath backpropagation affects optimization performance, but I don't know if it's right.

**Comment by haofeixu** [comment]
The motivation for doing this is to make the flow prediction before propagation unaffected by the subsequent results. However, I didn't observe too much difference by removing it as far as I can remember.

**Comment by mansang127** [comment]
> The motivation for doing this is to make the flow prediction before propagation unaffected by the subsequent results. However, I didn't observe too much difference by removing it as far as I can remember.

I get it, thank you.
